environment.yml

```diff
@@ -12,6 +12,8 @@ dependencies:
 - anaconda::setuptools=27.2.0
 - anaconda::statsmodels=0.8.0
 - conda-forge::altair=1.2.0
+- conda-forge::dask-searchcv
```

Comment:
Explicitly add `dask-searchcv` and `dask` to environment.yml both with versions. As far as importing dask in the notebook, only if it's necessary. Also consider using [anaconda dask](https://anaconda.org/anaconda/dask) unless that doesn't work. Currently, we're using the anaconda channel over conda-forge in terms of priority.